### DISCUSSION TITLE
Clear user when opening profiles

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20037] [Ideas]
In this case the user Orisa is undefined and the profile shown is from the last user

Sometimes it just takes long to load, sometimes I'm going to fast for it to load
Either way it's confusing

Would be nice to clear the data somehow and show a blank profile

https://user-images.githubusercontent.com/40531871/187555992-c7834951-3ed1-4bd9-af58-836c48befa94.mp4

**Comment by peppy** [comment]
Probably covered by https://github.com/ppy/osu/issues/11155 and other similar issues. The overlay needs some serious work.
